ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,284 @@
+  makeBooleanRaidBuffInput,
+  makeEnumIndividualBuffInput,
+  makeMultistateIndividualBuffInput,
+  makeMultistateRaidBuffInput,
+  makeTristateDebuffInput,
+  makeTristateIndividualBuffInput,
+  makeTristateRaidBuffInput,
+  withLabel
+} from "../icon_inputs";
+
+import * as InputHelpers from '../input_helpers';
+
+///////////////////////////////////////////////////////////////////////////
+//                                 RAID BUFFS
+///////////////////////////////////////////////////////////////////////////
+
+// TODO: Classic buff icon by level
+export const AllStatsBuff = withLabel(
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(9885), impId: ActionId.fromSpellId(17055), fieldName: 'giftOfTheWild'}),
+	'Mark of the Wild',
+);
+
+export const AllStatsPercentBuff = InputHelpers.makeMultiIconInput([
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(20217), fieldName: 'blessingOfKings', faction: Faction.Alliance}),
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(409580), fieldName: 'aspectOfTheLion'}),
+], 'Stats %');
+
+// TODO: Classic armor buff ranks
+export const ArmorBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(10293), impId: ActionId.fromSpellId(20142), fieldName: 'devotionAura', faction: Faction.Alliance}),
+	makeBooleanRaidBuffInput({id: ActionId.fromItemId(1478), fieldName: 'scrollOfProtection'}),
+], 'Armor');
+
+export const StaminaBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(10938), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude'}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(10937), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 48, maxLevel: 59}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(2791), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 36, maxLevel: 47}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(1245), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 24, maxLevel: 35}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(1244), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 12, maxLevel: 23}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(1243), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 1, maxLevel: 11}),
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(11767), impId: ActionId.fromSpellId(18696), fieldName: 'bloodPact'}),
+	makeBooleanRaidBuffInput({id: ActionId.fromItemId(10307), fieldName: 'scrollOfStamina'}),
+], 'Stamina');
+
+export const StrengthRaidBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(25361), impId: ActionId.fromSpellId(16295), fieldName: 'strengthOfEarthTotem', faction: Faction.Horde}),
+	makeBooleanRaidBuffInput({id: ActionId.fromItemId(10310), fieldName: 'scrollOfStrength'}),
```

Comment:
Make sure to merge in my changes from yesterday that add Horn of Lordaeron to the Str and Agi raid buff options, and move the scrolls to the Agi and Str consumes options instead. Thanks for putting in the faction filters!

Reply:
Aah thanks I merged `sod-fork` in earlier but must've missed that